Producer/Producer.Droid/Framework/Fragments/SettingsFragment.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Android.Views;
+using Android.OS;
+using Android.Content;
+using Producer.Domain;
+using Android.Preferences;
+using Android.Support.V4.Content;
+using SettingsStudio;
+using System.Collections.Generic;
+
+namespace Producer.Droid
+{
+	public class SettingsFragment : PreferenceFragment
+	{
+		static readonly string [] Preferences = { 
+			SettingsKeys.TestProducer,
+			SettingsKeys.MobileCenterKey,
+			SettingsKeys.RemoteFunctionsUrl,
+			SettingsKeys.RemoteDocumentDbUrl,
+			SettingsKeys.NotificationsName,
+			SettingsKeys.NotificationsConnectionString,
+			SettingsKeys.EmbeddedSocialKey,
+			SettingsKeys.UserReferenceKey
+		};
+
+		public override void OnCreate (Bundle savedInstanceState)
+		{
+			base.OnCreate (savedInstanceState);
+			SetHasOptionsMenu (true);
+			AddPreferencesFromResource (Resource.Xml.preferences);
+			FindPreference ("VersionNumberString").Summary = $"{Settings.VersionNumber} ({Settings.BuildNumber})";
```

Comment:
Roberto merge master into this branch then change this to use [`Settings.VersionDescription`](https://github.com/technicalpoets/producer/blob/master/Producer/Producer.Shared/Settings/SettingAccessors.cs#L13)